docs/source/tutorials/code_snippets/event_loop_qt.py

```diff
@@ -1,4 +1,4 @@
-from pyface.qt.QtCore import QApplication
+from pyface.qt.QtCore import QCoreApplication
```

Comment:
I think we still want to use the `QApplication` object. It possible got moved around between PyQt4 and PyQt5. You should be able to look for other uses of `QApplication` in the codebase to see where to import it from.